tcf_website/management/commands/load_semester.py

```diff
@@ -252,29 +252,49 @@ def load_section(
             units,
             section_type):
 
+        unique_params = {}
         params = {}
-        fields = {
+
+        unique_fields = {
             'sis_section_number',
             'semester',
+        }
+        fields = {
             'course',
             'topic',
             'units',
-            'section_type'}
+            'section_type'
+        }
+
         for k, v in locals().items():
```

Comment:
Since `key` and `value` are used below and the statements are short, I don't think there's harm in using `key` and `harm` instead of `k` and `v`.

Reply:
Changed in latest commit